data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -87,7 +87,7 @@
 )
 
 # Energy
-ENERGY_BURDEN_FIELD = "Energy burden"
+ENERGY_BURDEN_FIELD = "BURDEN"
```

Comment:
This can go back to `Energy burden` once you finish the renaming.

Reply:
Yup